### ISSUE TITLE
Fix runtime warnings

### ISSUE THREAD

**Issue by abir-taheer** [stuyspec/stuyspec.com#250]
Lots of runtime warnings show up during the build stage of CI. During CI, react treats these warnings as errors and thus the CI doesn't pass. Many of these are unused variables and images without alt text. More info is available [here](https://app.circleci.com/pipelines/github/stuyspec/stuyspec.com/20/workflows/dfaa2a40-dc55-43d8-850d-0711475c1d6b/jobs/1158) in the failing build step.
